packages/plugin-dev/src/node/hosted-instance-manager.ts

```diff
@@ -293,6 +293,10 @@ export abstract class AbstractHostedInstanceManager implements HostedInstanceMan
                 }
             };
 
+            if (isWindows) {
+                options.shell = true;
```

Comment:
Do you mind adding a comment on why this change is necessary?

```suggestion
                // Has to be set for running on windows (electron).
                // See also: https://github.com/nodejs/node/issues/3675
                options.shell = true;
```

Reply:
OK, I will amend this and PR again. 
Thank you very much!